runtime/server_http_request.go

```diff
@@ -142,6 +145,7 @@ func NewServerHTTPRequest(
 
 	req.res = NewServerHTTPResponse(w, req)
 	req.start()
+	req.setupLogFields(logFields)
```

Comment:
shift the call to start(). The log field setup will then be similar to response.finish()